nomad/server.go

```diff
@@ -1212,13 +1214,15 @@ func (s *Server) setupRpcServer(server *rpc.Server, ctx *RPCContext) {
 	eval := &Eval{srv: s, ctx: ctx, logger: s.logger.Named("eval")}
 	node := &Node{srv: s, ctx: ctx, logger: s.logger.Named("client")}
 	plan := &Plan{srv: s, ctx: ctx, logger: s.logger.Named("plan")}
+	serviceReg := &ServiceRegistration{srv: s, ctx: ctx}
 
 	// Register the dynamic endpoints
 	server.Register(alloc)
 	server.Register(deployment)
 	server.Register(eval)
 	server.Register(node)
 	server.Register(plan)
+	_ = server.Register(serviceReg)
```

Comment:
Nitpick: why have the `_ =` here, if we didn't do so above?

Reply:
I feel that if a return value is not being used or checked, we should explicitly discard the value. It makes it obvious the code has considered and understood the return and also stops my IDE lighting up with warnings.

In the future i'd like to change them all to this pattern along with numerous other places in the code base, but that is obviously out of scope here and would probably warrant internal discussions.